fetchers/balancer/index.ts

```diff
@@ -33,7 +33,7 @@ const balancerFetcher: Fetcher = {
       const prices: PriceDataFetched[] = [];
       for (const symbol in pricesObj) {
         prices.push({
-          symbol,
+          symbol: symbol,
```

Comment:
Why don't you like the shorter syntax? ;)

Reply:
I've changed this to "token" and than back to symbol (really can't decide between these to names)...shorter syntax restored..